### DISCUSSION TITLE
What's the relationship between the revm module and the evm module

### DISCUSSION THREAD

**Discussion by qingfengzxr** [paradigmxyz/reth#11704] [General]
Hello, friends. I don't know the relationship between the revm module and the evm module. If I want to learn about Rust implementations of EVM, which module should I look at?

**Comment by onbjerg** [comment]
The evm module in reth is a wrapper around revm, which is the actual implementation of the EVM

**Reply by qingfengzxr** [reply]
Got it, thank you for your quick response!
